public/docs/_examples/intro/dart/lib/logger_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.logger_service;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Injectable()
+class Logger {
```

Comment:
I wonder if we could/should use https://pub.dartlang.org/packages/logging here. @kevmoo?